
from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_CargoManagementWindow(object):
    def setupUi(self, CargoManagementWindow):
        CargoManagementWindow.setObjectName("CargoManagementWindow")
        CargoManagementWindow.resize(1067, 765)
        self.centralwidget = QtWidgets.QWidget(CargoManagementWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.layoutWidget2 = QtWidgets.QWidget(self.centralwidget)
        self.layoutWidget2.setGeometry(QtCore.QRect(440, 10, 351, 531))
        self.layoutWidget2.setObjectName("layoutWidget2")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.layoutWidget2)
        self.gridLayout_2.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.label_2 = QtWidgets.QLabel(self.layoutWidget2)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.gridLayout_2.addWidget(self.label_2, 0, 0, 1, 2)
        self.salesTableWidget = QtWidgets.QTableWidget(self.layoutWidget2)
        self.salesTableWidget.setObjectName("salesTableWidget")
        self.salesTableWidget.setColumnCount(0)
        self.salesTableWidget.setRowCount(0)
        self.gridLayout_2.addWidget(self.salesTableWidget, 1, 0, 1, 2)
        self.addAllButton = QtWidgets.QPushButton(self.layoutWidget2)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.addAllButton.setFont(font)
        self.addAllButton.setObjectName("addAllButton")
        self.gridLayout_2.addWidget(self.addAllButton, 2, 0, 1, 1)
        self.putButton = QtWidgets.QPushButton(self.layoutWidget2)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.putButton.setFont(font)
        self.putButton.setObjectName("putButton")
        self.gridLayout_2.addWidget(self.putButton, 2, 1, 1, 1)
        self.layoutWidget3 = QtWidgets.QWidget(self.centralwidget)
        self.layoutWidget3.setGeometry(QtCore.QRect(820, 10, 171, 531))
        self.layoutWidget3.setObjectName("layoutWidget3")
        self.gridLayout_3 = QtWidgets.QGridLayout(self.layoutWidget3)
        self.gridLayout_3.setContentsMargins(0, 0, 0, 0)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.returnpushButton = QtWidgets.QPushButton(self.layoutWidget3)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.returnpushButton.setFont(font)
        self.returnpushButton.setObjectName("returnpushButton")
        self.gridLayout_3.addWidget(self.returnpushButton, 0, 0, 1, 1)
        self.deleteButton = QtWidgets.QPushButton(self.layoutWidget3)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.deleteButton.setFont(font)
        self.deleteButton.setObjectName("deleteButton")
        self.gridLayout_3.addWidget(self.deleteButton, 1, 0, 1, 1)
        self.layoutWidget = QtWidgets.QWidget(self.centralwidget)
        self.layoutWidget.setGeometry(QtCore.QRect(30, 10, 371, 531))
        self.layoutWidget.setObjectName("layoutWidget")
        self.gridLayout = QtWidgets.QGridLayout(self.layoutWidget)
        self.gridLayout.setContentsMargins(0, 0, 0, 0)
        self.gridLayout.setObjectName("gridLayout")
        self.label = QtWidgets.QLabel(self.layoutWidget)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.gridLayout.addWidget(self.label, 0, 0, 1, 1)
        self.purchaseTableWidget = QtWidgets.QTableWidget(self.layoutWidget)
        self.purchaseTableWidget.setObjectName("purchaseTableWidget")
        self.purchaseTableWidget.setColumnCount(0)
        self.purchaseTableWidget.setRowCount(0)
        self.gridLayout.addWidget(self.purchaseTableWidget, 1, 0, 1, 2)
        self.updatepushButton = QtWidgets.QPushButton(self.layoutWidget)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.updatepushButton.sizePolicy().hasHeightForWidth())
        self.updatepushButton.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.updatepushButton.setFont(font)
        self.updatepushButton.setObjectName("updatepushButton")
        self.gridLayout.addWidget(self.updatepushButton, 2, 0, 1, 1)
        self.pushButton = QtWidgets.QPushButton(self.layoutWidget)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.pushButton.sizePolicy().hasHeightForWidth())
        self.pushButton.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setFamily("华文行楷")
        font.setPointSize(16)
        self.pushButton.setFont(font)
        self.pushButton.setText("")
        self.pushButton.setObjectName("pushButton")
        self.gridLayout.addWidget(self.pushButton, 2, 1, 1, 1)
        CargoManagementWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(CargoManagementWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 1067, 26))
        self.menubar.setObjectName("menubar")
        CargoManagementWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(CargoManagementWindow)
        self.statusbar.setObjectName("statusbar")
        CargoManagementWindow.setStatusBar(self.statusbar)

        self.retranslateUi(CargoManagementWindow)
        QtCore.QMetaObject.connectSlotsByName(CargoManagementWindow)

    def retranslateUi(self, CargoManagementWindow):
        _translate = QtCore.QCoreApplication.translate
        CargoManagementWindow.setWindowTitle(_translate("CargoManagementWindow", "MainWindow"))
        self.label_2.setText(_translate("CargoManagementWindow", "正在销售"))
        self.addAllButton.setText(_translate("CargoManagementWindow", "一键添加"))
        self.putButton.setText(_translate("CargoManagementWindow", "上架"))
        self.returnpushButton.setText(_translate("CargoManagementWindow", "返回主界面"))
        self.deleteButton.setText(_translate("CargoManagementWindow", "下架"))
        self.label.setText(_translate("CargoManagementWindow", "采购清单"))
        self.updatepushButton.setText(_translate("CargoManagementWindow", "更新"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    CargoManagementWindow = QtWidgets.QMainWindow()
    ui = Ui_CargoManagementWindow()
    ui.setupUi(CargoManagementWindow)
    CargoManagementWindow.show()
    sys.exit(app.exec_())

